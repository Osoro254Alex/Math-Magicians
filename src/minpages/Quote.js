import React from 'react';
import styles from './Quote.module.css';

const Quote = () => (
  <div className={styles.quote}>
    <ul>
      <li>
        I hope that seeing the excitement of solving this problem will make
        mathematicians realize that there are lots and lots of other problems in
        mathematics which are going to be just as challenging in the future. —
        <span>
          <big>
            <b>Andrew Wiles</b>
          </big>
        </span>
      </li>
      <li>
        If people do not believe that mathematics is simple, it is only because
        they do not realize how complicated life is. —
        <span>
          <big>
            <b>John von Neumann</b>
          </big>
        </span>
      </li>
    </ul>
  </div>
);

export default Quote;
