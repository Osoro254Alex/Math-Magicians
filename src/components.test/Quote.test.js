import React from 'react';
import { render, cleanup } from '@testing-library/react';
import renderer from 'react-test-renderer';
import Quote from '../components/Quote';

afterEach(cleanup);

describe('render welcome component without error', () => {
  test('welcome should render', () => {
    render(<Quote />);
  });
  test('matches snapshot for new Header', () => {
    const tree = renderer.create(<Quote />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
