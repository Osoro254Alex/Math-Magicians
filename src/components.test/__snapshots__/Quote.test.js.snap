// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`render welcome component without error matches snapshot for new Header 1`] = `
<div
  className="quote"
>
  <ul>
    <li>
      I hope that seeing the excitement of solving this problem will make mathematicians realize that there are lots and lots of other problems in mathematics which are going to be just as challenging in the future. —
      <span>
        <big>
          <b>
            Andrew Wiles
          </b>
        </big>
      </span>
    </li>
    <li>
      If people do not believe that mathematics is simple, it is only because they do not realize how complicated life is. —
      <span>
        <big>
          <b>
            John von Neumann
          </b>
        </big>
      </span>
    </li>
  </ul>
</div>
`;
